R/get_PITobs.R

```diff
@@ -36,6 +36,7 @@ get_PITobs = function(query_type = c('obs_site', 'release_site'),
   query_type <- match.arg(query_type)
 
   stopifnot(!is.null(start_date)|!is.null(end_date))
+  stopifnot(!is.na(as.Date(start_date, format ='%m/%d/%Y'))|!is.na(as.Date(end_date, format ='%m/%d/%Y')))
```

Comment:
I wonder if we could check the format of the dates supplied but not explicitly change them to dates?  An actually date value isn't needed for the url params.

Reply:
See next commit!